src/Provider.js

```diff
@@ -0,0 +1,33 @@
+import React, { createContext, useReducer } from 'react'
+
+export const Context = createContext()
+
+export const START = 'ASYNC_OPS.START'
+export const COMPLETE = 'ASYNC_OPS.COMPLETE'
+export const FAIL = 'ASYNC_OPS.ERROR'
+
+export const determineKey = ({ name, hookId, runId }) => [name, hookId, runId].filter(x => x).join('_')
```

Comment:
is this identity filter needed?

Reply:
I guess not.  It was there incase we didn't put a hookId or a runId but there's not really a reason for that.  Removing it.